examples/dataservice-read/example.cpp

```diff
@@ -34,12 +34,71 @@
 namespace {
 const std::string gKeyId("");      // your here.access.key.id
 const std::string gKeySecret("");  // your here.access.key.secret
-const std::string gCatalogHRN(
-    "hrn:here:data:::here-optimized-map-for-visualization-2");
-const std::string gLayerId("omv-base-v2");
-const std::string gPartitionId("100000093");  // Helsinki, Finland
+const std::string gCatalogHRN("hrn:here:data:::edge-example-catalog");
+const size_t gMaxLayers(5);
+const size_t gMaxPartitions(5);
+const char gLogLabel[] = "read-example";
 }  // namespace
 
+std::string handleCatalogResponse(
+    const olp::dataservice::read::CatalogResponse& catalogResponse) {
+  std::string firstPartitionId;
```

Comment:
```suggestion
std::string first_partition_id;
```

as mentioned in https://google.github.io/styleguide/cppguide.html#Variable_Names